packages/alfa-rules/src/sia-r18/rule.ts

```diff
@@ -57,6 +63,51 @@ export default Rule.Atomic.of<Page, Attribute>({
   },
 });
 
+function ariaHtmlAllowed(target: Attribute): boolean {
+  const attributeName = target.name as aria.Attribute.Name;
+  switch (target.owner.get().name) {
```

Comment:
```suggestion
  switch (target.owner.getOr("").name) {
```
Let's make this error-safe, if we somehow end up with an attribute with no parent this will trigger the default case and return false.

Actually, `target.owner.get()` is used in a couple other places (input and select), so we can refactor the whole as:
```typescript
for (const element of target.owner) {
  switch (element.name) {
    …
  }
}

return false;
```